front-end/src/components/Project.js

```diff
@@ -1,94 +1,105 @@
-import React, { useState, useEffect } from "react";
+import React, {useState, useEffect} from "react";
```

Comment:
We should probably change the file name from "Project.js" to something more relevant, like "SelectProject.js". I made this change on the branch 19-dashboard-ui, so it'll probably be reflected when it's merged into development.